tests/core/util/test_string_helpers.py

```diff
@@ -1,76 +1,8 @@
 # Test the helper objects in util.string.
 
-import base64 as stdlib_base64
 import re
 
-import pytest
-
-from core.util.string_helpers import UnicodeAwareBase64, base64, random_string
-
-
-class TestUnicodeAwareBase64:
-    def test_encoding(self):
-        string = "םולש"
-
-        # Run the same tests against two different encodings that can
-        # handle Hebrew characters.
-        self._test_encoder(string, UnicodeAwareBase64("utf8"))
-        self._test_encoder(string, UnicodeAwareBase64("iso-8859-8"))
-
-        # If UnicodeAwareBase64 is given a string it can't encode in
-        # its chosen encoding, an exception is the result.
-        shift_jis = UnicodeAwareBase64("shift-jis")
-        pytest.raises(UnicodeEncodeError, shift_jis.b64encode, string)
```

Comment:
Do we no longer need to test these encodings / exception or are they no longer applicable for some reason?

Reply:
I don't think this test was ever very meaningful, since we only ever created a `utf8` `UnicodeAwareBase64` object. We don't have a wrapper class now, just some wrapped functions that all use `utf8`, so it didn't seem meaningful to test other encoding that we don't use.

We could have tests that test `_wrap_func_bytes_string` with other encodings, but I don't know how helpful these tests are, since we're not using these encodings. If you think we need these tests I can add them here.